pennylane/operation.py

```diff
@@ -152,7 +152,7 @@ class ObservableReturnTypes(Enum):
 
     def __repr__(self):
         """String representation of the return types."""
-        return self.value
+        return str(self.value)
```

Comment:
Codefactor was complaining about this not being a string. I'm assuming there has been a new pylint release, and this is wasn't being caught before.